x-pack/plugins/security_solution/public/management/pages/response_actions/view/response_actions_list_page.test.tsx

```diff
@@ -671,26 +671,6 @@ describe('Response actions history page', () => {
       expect(history.location.search).toEqual('');
     });
 
-    it('should clear `actionTypes` selected options on `types` filter', () => {
```

Comment:
Why was this test removed?

Reply:
With the change of flag, this test has the very same outcome as the test below. Sorry I thought I added a comment explaining this.